src/olympia/lib/settings_base.py

```diff
@@ -707,6 +707,7 @@ def lazy_langs(languages):
         'zamboni/reviewers': (
             'css/zamboni/reviewers.styl',
             'css/zamboni/unlisted.less',
+            'css/zamboni/reviewers.less',
```

Comment:
mmm having one `reviewers.styl` and one `reviewers.less` file is going to be really annoying

Reply:
I couldn't bring myself to add anything to `reviewers.styl`, sorry.  For the total of **4** .styl files we should really convert them to .less or flatten to .css permanently and remove the dependency.

Reply:
#7815 for removing Stylus